### ISSUE TITLE
Public (pawtucket) theme test

### ISSUE THREAD

**Issue by jairomelo** [ImaginingFutures/imagining#18]

**Comment by jairomelo** [comment]
I found myself an error:

Places detail view only shows the last element in the array of elements. Example: https://testing.ifrepo.world/index.php/Detail/places/243

The point doesn't display a list of elements.

**Comment by Zabandosk** [comment]
labels and fields show the correct information
	objects--yes, no problem
	other detail pages don't have much information
	
urls are clickable and point correctly to its destination
	yes, no problem

images, documents, video and audio are displayed correctly
	images - yes, no problem
	pdf - yes, no problem
	video - NO, you cannot see the video. [Example](https://testing.ifrepo.world/index.php/Detail/objects/3)
	3d - not tested

error screens or partially rendered screens
	Number of cards/items you can see on browse screens (objects, participants and resources) is 36. And when you reach the bottom of the page, it is not loading other items, or it is not offering a "next" button.

	Related objects are not displayed as cards in entities page. E.g. [here](https://testing.ifrepo.world/index.php/Detail/entities/1)

	Home page->Empty white row between the footer and content.

	Details page->Next and preview buttons are looking useless grey areas. They have no function.

**Comment by jairomelo** [comment]
> no problem video - NO, you cannot see the video. [Example](https://testing.ifrepo.world/index.php/Detail/objects/3) 

It seems that the video you tested is not in an accepted format. "Media format: MTS"

In this example https://testing.ifrepo.world/index.php/Detail/objects/2 you can see that it works as intended.

**Comment by Zabandosk** [comment]
OK. I didn't notice. It was written as .mp4 in the file's general properties

**Comment by jairomelo** [comment]
> Related objects are not displayed as cards in entities page. E.g. [here](https://testing.ifrepo.world/index.php/Detail/entities/1)

That's a feature of CA when an entity has only one resource associated. For instance: https://testing.ifrepo.world/index.php/Detail/entities/52 
There you have cards display.

**Comment by jairomelo** [comment]
> Details page->Next and preview buttons are looking useless grey areas. They have no function.

Agree, but any attempt to remove those elements represent a complete change (deconfiguration) in the layout. It need to be fixed in a future iteration.

**Comment by jairomelo** [comment]
> error screens or partially rendered screens
> Number of cards/items you can see on browse screens (objects, participants and resources) is 36. And when you reach the bottom of the page, it is not loading other items, or it is not offering a "next" button.

Fixed!

To correct that issue:

- Update and upgrade server. Was also necessary to reboot to update the kernel.
